models/matrix.rb

```diff
@@ -19,6 +19,11 @@
 
 module Models
   class Matrix < Base
+    field :original_id
     fields :time, :distance, :value
+
+    def self.array_field_names
```

Comment:
Why "array", can't we do just `field_names`? Or `types` (like the type of the matrix -- `Models::Matrix.types`)?

Reply:
`field_names` is a method provided by active_hash which returns all the fields of the Model. Here, I only expect the proper matrices fields without the new `original_id` field. `types` is not expressive to me, do you have some other suggestions ?

Reply:
I can't think of a more expressive name than matrix "type" (like in time matrix, value matrix, etc, here time, distance, value is the type of the matrix). What about `matrix_field_names` -- i.e., the names of the fields which actually hold the matrices?

Reply:
Thanks for the precision on why you find "type" more expressive, It seems good to me now :)